### DISCUSSION TITLE
Code completion webstorm

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#11151] [Archive]
Did anyone add hints for hook contexts?

For example i want to hints for ItemService
![image](https://user-images.githubusercontent.com/36981278/150104642-575ca98b-4320-43f6-a39b-4d0b4a2ded0d.png)\

require from package is not working:
![image](https://user-images.githubusercontent.com/36981278/150104776-0b01cf14-7cec-4b58-8595-9991e5528dd7.png)

**Comment by Enhed** [comment]
`ItemsService` of course, but its nvm, hints not working with correct type names

**Comment by Enhed** [comment]
its was bug of webstorm

**Reply by NeoKactus** [reply]
Was there something you had to configure in webstorm? I have the same issue.
